src/lib/metadata.ts

```diff
@@ -0,0 +1,80 @@
+import { MetadataEntry, EncryptedMetadataManager } from "@saleor/app-sdk/settings-manager";
```

Comment:
Rebase with main, I added some metadata handler from different project and we can safely drop it. Sorry for that :c